tools/train_classifier.py

```diff
@@ -62,15 +56,40 @@ def clean_pipe_state_dict_key(key):
     path_annotations_valid = args['--path_annotations_valid'] or None
     num_layers_to_finetune = int(args['--num_layers_to_finetune'])
     temporal_training = args['--temporal_training']
+    resume = args['--resume']
+
+    # Create a folder "checkpoints" to save model weights
+    path = os.path.join(path_in, "checkpoints")
+    os.makedirs(path, exist_ok=True)
 
     # Load feature extractor
     feature_extractor = feature_extractors.StridedInflatedEfficientNet()
-    checkpoint = torch.load('resources/backbone/strided_inflated_efficientnet.ckpt')
+
+    if resume:
```

Comment:
i think you should not make the check on resume.
Check if the different files that will be created with the training ( label2int, checkpoints, ...) are already existing in the output folder

Reply:
> i think you should not make the check on resume.
> Check if the different files that will be created with the training ( label2int, checkpoints, ...) are already existing in the output folder

I'll move the warning outside resume. In addition, is the warning code ok? Or I have to check each file name (xxx.checkpoint, xxx.json, xxx.png, xxx.npy)?